tox.ini

```diff
@@ -21,18 +20,6 @@ commands =
     poetry install --no-root --sync --all-extras
     poetry run pytest {posargs}
 
-[testenv:pydantic1]
```

Comment:
Is this deliberately included? It doesn't seem to be related to the PR content from the description.

Reply:
Ok. I'll remove it and make from this PR.